src/main/java/gregtech/api/metatileentity/multiblock/MultiblockControllerBase.java

```diff
@@ -301,22 +304,59 @@ public List<IMultiblockPart> getMultiblockParts() {
         return Collections.unmodifiableList(multiblockParts);
     }
 
+    /**
+     * Grabs the current tier of the multi(whether in DummyWorld or Server) useful for whatever you want
+     */
+    public int getTier() {
+        return this.structureTier;
+    }
+
+    /**
+     * sets the tier of the multi clamped between 1 and maxTier()
+     */
+    public void setTier(int structureTier) {
+        if(this.structureTier != structureTier) {
+            this.structureTier = MathHelper.clamp(structureTier, 0, getMaxTier());
+            if (getWorld() != null && !getWorld().isRemote) {
+                reinitializeStructurePattern();
+            }
+        }
+    }
+
+    /**
+     *  Override if you are using the multiblock tiered system
+     *  max tier of 0 means tiering is disabled
+     */
+    public int getMaxTier() {
+        return 0;
+    }
```

Comment:
these methods should be called `structure tier` not `tier`, too general for this low down of methods